spec/models/provider_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Provider, type: :model do
+  let(:subject) { create(:provider) }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] RSpec/EmptyLineAfterFinalLet: Add an empty line after the last let.